### ISSUE TITLE
Update readthedocs for mixbox

### ISSUE THREAD

**Issue by ikiril01** [MAECProject/python-maec#75]
The current readthedocs documentation doesn't take into account the mixbox related changes, and so it should be updated accordingly.

**Comment by bworrell** [comment]
@ikiril01, this should wait until we are ready to push out a mixboxified python-maec release.

**Comment by ikiril01** [comment]
@bworrell yup, agree 100%! I should have clarified this point in the issue description :)

**Comment by gtback** [comment]
The "default" in all of the RTD pages should be "stable", not "latest". It's possible we should add a warning in master, which will get picked up if people try to look at the "latest" docs.
